.github/automation/aarch64/git_bisect.sh

```diff
@@ -0,0 +1,35 @@
+#! /bin/bash
+
+# *******************************************************************************
+# Copyright 2025 Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# *******************************************************************************
+
+# Usage: ./git_bisect.sh GOOD_SHA BAD_SHA BUILD_DIR BENCHDNN_CMD
+set -o pipefail
+
+GOOD=$1
+BAD=$2
+BUILD_DIR=$3
+CMD=${@:4}
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+ONEDNN_TEST_SET=NIGHTLY ONEDNN_BUILD_GRAPH=OFF ${SCRIPT_DIR}/build.sh --configure
```

Comment:
The [correct option seems to be](https://github.com/uxlfoundation/oneDNN/blob/f4dd76bda1a5660b9ca8143a307539ac17c96164/.github/automation/aarch64/build.sh#L35) `configure` not `--configure`.